### ISSUE TITLE
Asking about the Mask-Rcnn training strategy 

### ISSUE THREAD

**Issue by vicchu** [jiwoon-ahn/irn#11]
Hi, Jiwoon Ahn
After transforming the  pseudo label  to the COCO-style annotations, I trained  the Mask R-CNN with ResNet-50-FPN .

But the performance i got is slightly lower than the report ，mAP50 is 45.0.

![image](https://user-images.githubusercontent.com/38305523/62544479-e3404a00-b870-11e9-8b5f-376c4935cee5.png)

I 'd like to ask you about the mask-rcnn training strategy, what kind data augmentation you adopt.

Thank you !

**Comment by jiwoon-ahn** [comment]
Hi @vicchu
You may try with smaller image size and smaller training steps.
> SOLVER:
>   WEIGHT_DECAY: 0.0001
>   LR_POLICY: steps_with_decay
>   BASE_LR: 0.01
>   GAMMA: 0.1
>   MAX_ITER: 18000 
>   STEPS: [0, 12000, 16000]
> TRAIN:
>   WEIGHTS: https://s3-us-west-2.amazonaws.com/detectron/ImageNetPretrained/MSRA/R-50.pkl
>   DATASETS: ('voc_2012_train',)
>   SCALES: (512,)
>   MAX_SIZE: 800
>   BATCH_SIZE_PER_IM: 512
>   RPN_PRE_NMS_TOP_N: 2000  # Per FPN level
>   IMS_PER_BATCH: 4
> TEST:
>   DATASETS: ('voc_2012_val',)
>   SCALE: 512
>   MAX_SIZE: 800
>   NMS: 0.5
>   RPN_PRE_NMS_TOP_N: 1000  # Per FPN level
>   RPN_POST_NMS_TOP_N: 1000
> OUTPUT_DIR: .

**Comment by vicchu** [comment]
Hi @jiwoon-ahn 
After try the parameters you offered ,I still can not get the same result.
But i have try larger image size,the result is closer to what the paper report.
